src/utils.js

```diff
@@ -64,6 +64,83 @@ CKEDITOR.define( function() {
 		 */
 		isObject: function( obj ) {
 			return typeof obj === 'object' && !!obj;
-		}
+		},
+
+		/**
+		 * A mixin function to be used to implement the static `extend()` method in classes. It allows for easy creation
+		 * of subclasses.
```

Comment:
Code samples needed. It's important that we understand how you want it to be used and e.g. the `constructor` part can be incorrectly understood. This can be done together with https://github.com/ckeditor/ckeditor5-core/pull/13/files#r24818601.